aws/cloudformation/ecs/posthog.yaml

```diff
@@ -216,20 +218,30 @@ Parameters:
     Type: String
     NoEcho: true
     Default: posthogadmin
-
   RDSRestoreDBSnapshotId:
     Description: RDS snapshot to restore from. Read more at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html on updating DB instances
     Type: String
     Default: ''
-
   RDSPubliclyAvailable:
     Description: Make RDS instance available on the public web. Note that changing this on an already launched stack will create a new database from scratch.
     Type: String
     Default: false
     AllowedValues:
       - true
       - false
-
+  RDSBackupRetentionPeriod:
```

Comment:
I think this is valuable in its own right, but not for the database restore purposes. As mentioned under https://github.com/PostHog/deployment/issues/57 we automatically snapshot during replacement/deletion and users can restore the DB from that.

Reply:
Agreed!